common-arcana.lic

```diff
@@ -518,16 +518,17 @@ module DRCA
     end
   end
 
-  def update_astral_data(data)
+  def update_astral_data(data, settings)
     if data['moon']
       data = set_moon_data(data)
     elsif data['stats']
-      data = set_planet_data(data)
+      data = set_planet_data(data, settings)
     end
     data
   end
 
-  def find_visible_planets(planets)
+  def find_visible_planets(planets, settings)
+    return unless settings.have_telescope
     return if DRC.bput('get my telescope', 'You get', 'What were you', 'You are already') == 'What were you'
```

Comment:
💡 Go ahead and update the "getting" logic to use the equivalent DRCMM method, too
```suggestion
    DRCMM.get_telescope(settings.telescope_storage)
    return unless DRCI.in_hands?('telescope')
```